vm-systemd/qubes-network.service

```diff
@@ -8,6 +8,7 @@ After=network-pre.target qubes-iptables.service
 Type=oneshot
 RemainAfterExit=yes
 ExecStart=/usr/lib/qubes/init/network-proxy-setup.sh
+ExecStop=/usr/lib/qubes/init/network-proxy-stop.sh
```

Comment:
What happens if this script fails due to a transient error?  This is unlikely, but we should ensure that the firewall is not torn down until forwarding has definitively been disabled.  If disabling forwarding fails, the firewall needs to stay up.